recipes/psqlodbc/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = 'psqlODBC' %}
+{% set version = '11.01' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-{{ version }}.0000.tar.gz'
+  sha256: c5f1a6e472893f135a434f1276dffa5d64508f9d4fa37bb2e492fd0ad4b6042c
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python >=3.6
+  host:
+    - unixodbc
+    - postgresql
+  run:
+    - unixodbc
+    - libpq
+
+test:
+  commands:
+    - conda inspect linkages psqlodbc
+    - test -f $PREFIX/lib/psqlodbca.so  # [unix]
+    - test -f $PREFIX/lib/psqlodbcw.so  # [unix]
```

Comment:
Shouldn't this be `test -f $PREFIX/lib/psqlodbcw${SHLIB_EXT}`?, without the selector?

Reply:
No, this is correct. They are really using `.so` even on OSX, so we need the selector for the case when someone adds Windows support.

Reply:
Verified that this is the same in Homebrew.

Reply:
Weird.  A comment about this might be nice